internal/kubernetes/operator/dbusers/dbusers_test.go

```diff
@@ -194,10 +193,13 @@ func TestBuildDBUsers(t *testing.T) {
 				},
 			},
 			Spec: akov2.AtlasDatabaseUserSpec{
-				Project: akov2common.ResourceRefNamespaced{
+				Project: &akov2common.ResourceRefNamespaced{
 					Name:      resources.NormalizeAtlasName(projectName, dictionary),
 					Namespace: targetNamespace,
 				},
+				ExternalProjectRef: &akov2.ExternalProjectReference{
+					ID: projectID,
+				},
 				DatabaseName:    user.DatabaseName,
```

Comment:
This an invalid CRD, you can't have both internal and external project ref set.

Reply:
This is not the end result.

Reply:
This has been fixed, the reference is set properly as the flag is passed in to the resource builders.